<p><strong>Mcm10 Regulates DNA Replication Elongation by </strong></p>
<p><strong>Stimulating the CMG Replicative Helicase</strong></p>
<p>eesti</p>
<p>Marko Lõoke<sup>1</sup>*, Michael F. Maloney,<sup>1,2</sup>* and Stephen P. Bell<sup>1‡</sup></p>
<p>lõokeΓΔβ</p>
<p><sup>1</sup>Howard Hughes Medical Institute, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139 USA</p>
<p><sup>2</sup>Microbiology Graduate Program, Massachusetts Institute of Technology, Cambridge, MA 02139 USA</p>
<p>*these authors contributed equally to this manuscript</p>
<p><sup>‡</sup>Corresponding author:</p>
<p>email: spbell@mit.edu</p>
<p>phone: 617-253-2054</p>
<p>Running title: Mcm10 Stimulates Replication Elongation</p>
<p>Keywords: Cdc45/Mcm2-7/GINS, cell cycle, DNA replication fork, <em>S. cerevisiae</em>, reconstituted DNA replication</p>
<p><strong>ABSTRACT</strong></p>
<p>Activation of the Mcm2-7 replicative DNA helicase is the committed step in eukaryotic DNA replication initiation. Although Mcm2-7 activation requires binding of the helicase-activating proteins, Cdc45 and GINS (forming the CMG complex), an additional protein, Mcm10, drives initial origin DNA unwinding by an unknown mechanism. We show that Mcm10 binds a conserved motif located between the OB-fold and A-subdomain of Mcm2. Although buried in the interface between these domains in Mcm2-7 structures, mutations predicted to separate the domains and expose this motif restore growth to conditional-lethal <em>MCM10</em> mutant cells. We found that in addition to stimulating initial DNA unwinding, Mcm10 stabilizes Cdc45 and GINS association with Mcm2-7 and stimulates replication elongation <em>in vivo</em> and <em>in vitro</em>. Furthermore, we identify a lethal allele of <em>MCM10</em> that stimulates initial DNA unwinding but is defective in replication elongation and CMG binding. Our findings expand the roles of Mcm10 during DNA replication and suggest a new model for Mcm10 function as an activator of the CMG complex throughout DNA replication.</p>
<p><strong>INTRODUCTION</strong></p>
<p>Eukaryotic DNA replication initiation requires the sequential assembly of protein complexes at origins of replication. In G1 phase, the Mcm2-7 replicative helicase is loaded onto double-stranded DNA (dsDNA) as a head-to-head double hexamer in an inactive state <span id="__Fieldmark__36_3205866301" class="anchor"></span>(<span id="__Fieldmark__35_3683559902" class="anchor"></span>E<span id="__Fieldmark__129_3694128450" class="anchor"></span>vrin et al. 2009; Ticau et al. 2017; 2015; Remus et al. 2009). As cells progress into S phase, two kinases, S-phase cyclin-dependent kinase (S-CDK) and the Dbf4-dependent Cdc7 kinase (DDK), promote the association of two helicase activators, Cdc45 and GINS, with Mcm2-7. DDK phosphorylation of Mcm2-7 stimulates the association of Cdc45, Sld3 and Sld7 <span id="__Fieldmark__47_3205866301" class="anchor"></span></p>
<p>(Deegan et al. 2016; Heller et al. 2011)</p>
<p>followed by the S-CDK-dependent recruitment of a complex between Sld2, Dpb11, Pol <strong>ε</strong> and GINS <span id="__Fieldmark__58_3205866301" class="anchor"></span>(<span id="__Fieldmark__53_3683559902" class="anchor"></span>M<span id="__Fieldmark__160_3694128450" class="anchor"></span>uramatsu et al. 2010; Yeeles et al. 2015; Tanaka et al. 2007; Zegerman and Diffley 2007). Cdc45 and GINS association with Mcm2-7 forms the replicative DNA helicase, the Cdc45/Mcm2-7/GINS (CMG) complex <span id="__Fieldmark__69_3205866301" class="anchor"></span>(<span id="__Fieldmark__60_3683559902" class="anchor"></span>M<span id="__Fieldmark__174_3694128450" class="anchor"></span>oyer et al. 2006; Ilves et al. 2010), but initial DNA unwinding by this assembly and commitment to replication initiation requires Mcm10 <span id="__Fieldmark__80_3205866301" class="anchor"></span></p>
<p>(van Deursen et al. 2012; Kanke et al. 2012; Watase et al. 2012)</p>
<p>. The resulting single-stranded DNA (ssDNA) facilitates recruitment of the remaining DNA synthesis machinery <span id="__Fieldmark__88_3205866301" class="anchor"></span>(<span id="__Fieldmark__75_3683559902" class="anchor"></span>H<span id="__Fieldmark__200_3694128450" class="anchor"></span>eller et al. 2011).</p>
<p>The process of helicase activation requires the loaded Mcm2-7 double hexamer and its associated DNA to undergo major conformational changes. Initially, Mcm2-7 double hexamers encircle dsDNA <span id="__Fieldmark__101_3205866301" class="anchor"></span>(<span id="__Fieldmark__84_3683559902" class="anchor"></span>R<span id="__Fieldmark__214_3694128450" class="anchor"></span>emus et al. 2009; Evrin et al. 2009). In contrast, activated CMG complexes at replication forks contain a single Mcm2-7 complex and encircle ssDNA <span id="__Fieldmark__112_3205866301" class="anchor"></span>(<span id="__Fieldmark__91_3683559902" class="anchor"></span>F<span id="__Fieldmark__225_3694128450" class="anchor"></span>u et al. 2011; Yardimci et al. 2012; Sun et al. 2015; Georgescu et al. 2017). Structural studies have captured Mcm2-7 at multiple stages during helicase loading and in the CMG complex <span id="__Fieldmark__123_3205866301" class="anchor"></span></p>
<p>(Yuan et al. 2016; Li et al. 2015; Abid Ali et al. 2016; Sun et al. 2013; Georgescu et al. 2017)</p>
<p>. These structures have provided important insights into Mcm2-7 loading and the interactions of Mcm2-7 with Cdc45 and GINS. Nevertheless, the Mcm2-7 conformational changes necessary for DNA unwinding are controversial <span id="__Fieldmark__131_3205866301" class="anchor"></span></p>
<p>(Yuan et al. 2016; Abid Ali et al. 2016; Georgescu et al. 2017)</p>
<p>and the events that drive the transition from the initially-loaded, inactive Mcm2-7 double hexamer to the activated CMG complex are poorly understood.</p>
<p>Although the general consequences of Mcm10 loss are understood, how it activates the CMG complex to initiate DNA unwinding is unclear. Recruitment of Cdc45 or GINS to Mcm2-7 is independent of Mcm10 <span id="__Fieldmark__141_3205866301" class="anchor"></span></p>
<p>(van Deursen et al. 2012; Watase et al. 2012; Kanke et al. 2012; Yeeles et al. 2015)</p>
<p>. In contrast, Mcm10 is required for initial DNA unwinding at origins of replication <span id="__Fieldmark__149_3205866301" class="anchor"></span></p>
<p>(van Deursen et al. 2012; Watase et al. 2012; Kanke et al. 2012; Yeeles et al. 2015)</p>
<p>and has been implicated in the separation of the Mcm2-7 double hexamer <span id="__Fieldmark__158_3205866301" class="anchor"></span>(<span id="__Fieldmark__133_3683559902" class="anchor"></span>Q<span id="__Fieldmark__298_3694128450" class="anchor"></span>uan et al. 2015). It is possible that the double hexamer of the Mcm2-7 complex inhibits DNA unwinding and Mcm10 activates unwinding by causing double-hexamer separation <span id="__Fieldmark__169_3205866301" class="anchor"></span>(<span id="__Fieldmark__140_3683559902" class="anchor"></span>Q<span id="__Fieldmark__305_3694128450" class="anchor"></span>uan et al. 2015). Alternatively, Mcm10 could facilitate extrusion of ssDNA from the Mcm2-7 central channel, enabling the transition from Mcm2-7 encircling dsDNA to ssDNA <span id="__Fieldmark__180_3205866301" class="anchor"></span>(<span id="__Fieldmark__147_3683559902" class="anchor"></span>C<span id="__Fieldmark__311_3694128450" class="anchor"></span>osta et al. 2014). Finally, Mcm10 binding could directly activate CMG DNA unwinding, indirectly leading to the separation of the Mcm2-7 hexamers.</p>
<p>Several lines of evidence suggest that Mcm10 acts by interacting with Mcm2-7. Although unrelated to the Mcm2-7 proteins, Mcm10 binds to the Mcm2, Mcm4 and Mcm6 subunits of Mcm2-7 <span id="__Fieldmark__193_3205866301" class="anchor"></span></p>
<p>(Quan et al. 2015; Douglas and Diffley 2016)</p>
<p>. In addition, genetic studies suggest an important interaction between Mcm10 and Mcm2 <span id="__Fieldmark__201_3205866301" class="anchor"></span>(<span id="__Fieldmark__164_3683559902" class="anchor"></span>L<span id="__Fieldmark__343_3694128450" class="anchor"></span>ee et al. 2010; Homesley et al. 2000; Apger et al. 2010). Finally, Mcm10 associates with the replisome under certain conditions <span id="__Fieldmark__212_3205866301" class="anchor"></span>(<span id="__Fieldmark__171_3683559902" class="anchor"></span>G<span id="__Fieldmark__349_3694128450" class="anchor"></span>ambus et al. 2006; Ricke and Bielinsky 2004), although the biological significance of this interaction is unclear. Despite these observations, a specific Mcm10-binding site has not been identified on any Mcm2-7 subunit.</p>
<p>In this study, we use a combination of molecular genetics and reconstituted DNA replication assays to investigate Mcm10 function. Using Mcm2-Mcm10 interaction data, we identify a conserved Mcm10-binding motif in Mcm2. Although obscured in current Mcm2-7 structures, mutants designed to expose the Mcm10-binding motif bypassed conditional-lethal <em>MCM10</em> mutations. Consistent with a direct effect of Mcm10 binding on Mcm2-7, Mcm10 stabilized Cdc45 and GINS association with Mcm2-7. Additionally, we observe that Mcm10 stimulated replication elongation both <em>in vivo</em> and <em>in vitro</em> and characterize an Mcm10 separation-of-function mutant that is specifically defective in this elongation function. Our findings expand the roles of Mcm10 and illuminate its mechanism of function.</p>
<p><strong>RESULTS</strong></p>
<p><strong>Mcm10 binds a conserved region in the Mcm2 N-terminal domain</strong></p>
<p>During Mcm10 purification, we observed three co-purifying proteins. Two of the proteins co-migrated with Mcm6 and Mcm4 during SDS-PAGE (Fig. 1A). Consistent with recent findings (Douglas and Diffley, 2016; Quan et al., 2015), mass spectrometry confirmed these proteins were Mcm4 and Mcm6 and identified Mcm2 as the third protein (Table S1).</p>
<p>To understand the target of Mcm10 in more detail, we sought to identify the binding site for Mcm10 on Mcm2, Mcm4, or Mcm6. Consistent with prior studies that identified <em>MCM2</em> mutants as suppressors of <em>mcm10-1</em> <span id="__Fieldmark__242_3205866301" class="anchor"></span>(<span id="__Fieldmark__197_3683559902" class="anchor"></span>L<span id="__Fieldmark__445_3694128450" class="anchor"></span>ee et al. 2010), we found that Mcm10 showed robust interactions with Mcm2 and much weaker interactions with Mcm4 and Mcm6 (Fig. 1B). No binding to Mcm5 or Mcm7 was detected. Thus, we focused on localizing the strong Mcm10 binding site on Mcm2. All Mcm2-7 subunits include three folded domains, the A-subdomain, the oligonucleotide/oligosaccharide-fold (OB-fold) and the C-terminal AAA+ ATPase domain <span id="__Fieldmark__254_3205866301" class="anchor"></span>(<span id="__Fieldmark__205_3683559902" class="anchor"></span>L<span id="__Fieldmark__477_3694128450" class="anchor"></span>i et al. 2015). Testing Mcm10 binding to truncated forms of Mcm2 (Fig. 1C and 1D) showed that the A-subdomain, but not the OB-fold or AAA+ domains, bound Mcm10. Importantly, Mcm10 binding to Mcm2 required residues 290-299 of the A-subdomain, and mutating these residues in a larger Mcm2 fragment eliminated Mcm10 binding (Fig. 1D, lanes 19 and 20). Importantly, the region of Mcm2 bound by Mcm10 (hereafter, the Mcm10-binding motif) is highly conserved across eukaryotic species (Fig. 1Ei), but is absent in the other Mcm2-7 subunits (Fig. 1Eii), strongly suggesting that Mcm10 binding to Mcm2 is conserved.</p>
<p>We next tested the importance of this Mcm10-binding motif for Mcm2 function. When present as the only copy of the <em>MCM2</em> gene, yeast strains lacking (<em>mcm2-Δ290-299</em>) or with substitution mutations (<em>mcm2-mbm</em>) in the Mcm10-binding motif in Mcm2 showed strong growth defects or cell death, respectively (Fig. 1F, 5-FOA panel). These mutations are not dominant as normal cell growth is detected when wild-type (WT) <em>MCM2</em> is also present (Fig. 1F, -URA).</p>
<p><strong>Disrupting interactions in the Mcm2 N-terminal domain bypasses Mcm10 depletion</strong></p>
<p>The structures of the initially-loaded Mcm2-7 complex <span id="__Fieldmark__286_3205866301" class="anchor"></span>(<span id="__Fieldmark__233_3683559902" class="anchor"></span>L<span id="__Fieldmark__604_3694128450" class="anchor"></span>i et al. 2015) and the CMG complex <span id="__Fieldmark__297_3205866301" class="anchor"></span></p>
<p>(Yuan et al. 2016)</p>
<p>showed that the Mcm10-binding motif in Mcm2 is buried between the A-subdomain and the OB-fold of Mcm2, restricting the accessibility of these residues (Fig. 2A). This finding suggests that either Mcm10 captures or induces the displacement of the A-subdomain to access the Mcm10-binding motif. To investigate the importance of the interaction between the Mcm2 OB-fold and A-subdomain, we generated mutants at the interface between these domains (Fig. 2B). Each of these alleles was viable when present as the only copy of <em>MCM2</em> (Fig. 2C).</p>
<p>To test the hypothesis that Mcm10 displaces the Mcm2 A-subdomain, we asked if the mutations at the interface of the Mcm2 OB-fold and A-subdomain complemented the lethal depletion of Mcm10 from the nucleus. We used the anchor-away method <span id="__Fieldmark__309_3205866301" class="anchor"></span>(<span id="__Fieldmark__252_3683559902" class="anchor"></span>H<span id="__Fieldmark__666_3694128450" class="anchor"></span>aruki et al., 2008) to deplete Mcm10 linked to a rapamycin-binding protein (Mcm10-FRB) from the nucleus (Fig. 2D, top). Importantly, yeast strains containing <em>mcm10-FRB</em> showed rapamycin-dependent cell death that was rescued by expressing WT <em>MCM10</em> (Fig. 2D, rows 1 and 2). For most of the <em>MCM2</em> mutants, depletion of Mcm10 from the nucleus remained lethal. Remarkably, two of the mutants (<em>mcm2-bom1</em> [bypass of Mcm10] and <em>mcm2-bom2</em>) restored viability to cells depleted of Mcm10 (Fig. 2D). Supporting the hypothesis that disrupting the OB-fold/A-subdomain interaction complements Mcm10 depletion, the residues mutated in Mcm2-bom1 and Mcm2-bom2 are located opposite from one another on the OB-fold and A-subdomain, respectively.</p>
<p>To further explore the ability of the <em>mcm2-bom1</em> and <em>mcm2-bom2</em> alleles to bypass Mcm10 function, we tested two other <em>MCM10</em> conditional-lethal alleles, <em>mcm10-1</em> <span id="__Fieldmark__344_3205866301" class="anchor"></span><em>(</em><span id="__Fieldmark__283_3683559902" class="anchor"></span><em>H</em><span id="__Fieldmark__743_3694128450" class="anchor"></span><em>omesley et al. 2000; Merchant et al. 1997)</em> and <em>mcm10-1td</em> <span id="__Fieldmark__357_3205866301" class="anchor"></span><em>(</em><span id="__Fieldmark__292_3683559902" class="anchor"></span><em>v</em><span id="__Fieldmark__750_3694128450" class="anchor"></span><em>an Deursen et al. 2012)</em> and a complete <em>MCM10</em> deletion (<em>Δmcm10</em>). Under conditions that are lethal for <em>mcm10-1 and mcm10-1td</em>, we found that <em>mcm2-bom1</em> or <em>mcm2-bom2</em> restored cell viability (Fig. S1A and S1B). Despite restoring growth to the conditional-lethal alleles, <em>Δmcm10</em> could not be bypassed by either <em>mcm2-bom</em> or <em>mcm2-bom2</em> (Fig. S1C). The inability to bypass <em>Δmcm10</em> suggests that <em>mcm2-bom1</em> and <em>mcm2-bom2</em> require <em>MCM10</em>, but at much lower levels than WT <em>MCM2</em> (see Discussion). Consistent with this hypothesis, further depletion of Mcm10-1td protein (by induction of the Ubr1 protein) led to reduced growth rates in the presence of <em>mcm2-bom1</em> and <em>mcm2-bom2</em> (compare Fig. S1Bi and Bii). Nevertheless, the ability to rescue multiple conditional-lethal alleles of <em>MCM10</em> by mutating the Mcm2 OB-fold and A-subdomain interface strongly supports the conclusion that binding to this region of Mcm2 is critical for Mcm10 function.</p>
<p><strong>Mcm10 stabilizes the CMG complex</strong></p>
<p>We used a modified reconstituted DNA replication assay <span id="__Fieldmark__407_3205866301" class="anchor"></span>(<span id="__Fieldmark__338_3683559902" class="anchor"></span>Y<span id="__Fieldmark__843_3694128450" class="anchor"></span>eeles et al. 2015) to further investigate the mechanism and importance of the Mcm10-Mcm2 interaction during DNA replication. To mimic the <em>in vivo</em> order of replication events, we sequentially incubated subsets of purified replication proteins (Fig. S2A) with a replication-origin-containing, circular DNA template coupled to magnetic beads. This assay has many hallmarks of eukaryotic DNA replication, including dependence on the S-CDK and DDK kinases and all of the helicase-activating proteins (Fig. S2B). The polymerases and accessory DNA replication proteins used in the assay were also shown to be functional (Fig. S2C and S2D)</p>
<p>We initially assessed the requirements for Mcm10 binding during CMG formation, as it is controversial whether only Mcm2-7 loading <span id="__Fieldmark__424_3205866301" class="anchor"></span>(<span id="__Fieldmark__351_3683559902" class="anchor"></span>v<span id="__Fieldmark__895_3694128450" class="anchor"></span>an Deursen et al. 2012; Wohlschlegel et al. 2002; Karnani and Dutta 2011) or full CMG formation <span id="__Fieldmark__435_3205866301" class="anchor"></span></p>
<p>(Kanke et al. 2012; Watase et al. 2012; Heller et al. 2011; Douglas and Diffley 2016)</p>
<p>is required for recruitment of Mcm10 to origin DNA. We assembled CMG complexes using a simplified assay involving three steps: Mcm2-7 loading, DDK phosphorylation and CMG formation (Fig. 3A). Pol α, Pol δ and all nucleotides except ATP were omitted from the final step, allowing CMG formation and activation (Fig. 3B) but preventing DNA synthesis. We measured Mcm10 association with the DNA template after each step of the assay. Mcm10 did not associate with DNA alone or with loaded Mcm2-7 in the absence of DDK-treatment (Fig. 3A). DDK phosphorylation of loaded Mcm2-7 resulted in detectable Mcm10 binding but only at high Mcm10 concentrations. Importantly, Mcm10 showed ~10-fold higher affinity for the CMG complex relative to DDK-phosphorylated Mcm2-7 (Fig. 3A). Thus, both DDK-phosphorylation of Mcm2-7 and CMG complex formation contribute to Mcm10 recruitment.</p>
<p>We next evaluated the role of Mcm10 in CMG formation and activation. Consistent with previous findings, Mcm10 was required for the recruitment of the ssDNA binding protein, RPA, a marker for DNA unwinding (Fig. 3B; <span id="__Fieldmark__451_3205866301" class="anchor"></span></p>
<p>(van Deursen et al. 2012; Watase et al. 2012; Yeeles et al. 2015)</p>
<p>. Also in agreement with previous data <span id="__Fieldmark__460_3205866301" class="anchor"></span>(<span id="__Fieldmark__383_3683559902" class="anchor"></span>K<span id="__Fieldmark__979_3694128450" class="anchor"></span>anke et al. 2012; van Deursen et al. 2012; Heller et al. 2011; Yeeles et al. 2015), we found similar levels of DNA-associated Cdc45 and GINS regardless of the presence of Mcm10 after washing with a low-salt buffer (Fig. 3B, lanes 2 and 3). Washing the same reactions with a stringent high-salt buffer (containing 0.5 M NaCl) revealed that only CMG complexes treated with Mcm10 were retained on the DNA (Fig. 3B, lanes 5 and 6), while Pol <strong>ε</strong> and RPA were released. Interestingly, this increased stability of Cdc45 and GINS association did not require continued Mcm10 binding, as the high-salt wash also released Mcm10 from the template. Together, these data show that Mcm10 associates with and alters the CMG in a manner that stabilizes Cdc45 and GINS association with Mcm2-7.</p>
<p>To determine if the high-salt washed CMG complexes were true intermediates in the replication-initiation process and competent for DNA replication, we added replication-elongation proteins (Pol ε, Pol α, Pol δ, Top2, Ctf4, RPA, RFC, PCNA and Mcm10 [as indicated]) and all nucleotides to initiate DNA synthesis (Fig. 3C). Because free Cdc45 and GINS were removed during the high-salt wash, new CMG formation was prevented during this last incubation. No DNA synthesis was observed when Mcm10 or DDK was omitted from these reactions (Fig. 3C). When Mcm10 and DDK were included, DNA synthesis initiated from the high-salt-resistant CMG complexes (Fig. 3C, lane 6), indicating that they are functional replication intermediates. The reduced DNA replication initiating from the high-salt washed relative to low-salt washed CMG complexes (Fig. 3C, lanes 3 and 6) was likely caused by the higher amounts of Cdc45 and GINS retained after the low-salt wash (Fig. 3B) that were subsequently activated by Mcm10 present during the final DNA replication step.</p>
<p><strong>Mcm2 mutants that bypass Mcm10 function increase replication-product lengths </strong></p>
<p>To further explore the significance of the Mcm10-Mcm2 interactions <em>in vitro</em>, we purified Mcm2-7/Cdt1 complexes containing the <em>mcm2-mbm</em>, <em>mcm2-bom1</em> or <em>mcm2-bom2</em> mutations. We compared the WT and mutant complexes in the CMG formation assay followed by a low-salt wash. We detected significantly weaker binding of Mcm10 to CMG complexes formed with Mcm2-7<sup>2-mbm</sup> and Mcm2-7<sup>2-bom2</sup> (Fig. 3D), consistent with these mutants altering the Mcm10-binding motif. In contrast, Mcm10 association with Mcm2-7<sup>2-bom1</sup>, which does not alter the Mcm10-binding motif, was near WT levels. In addition to Mcm10-binding defects, Mcm2-7<sup>2-mbm</sup> and Mcm2-7<sup>2-bom2</sup> mutant complexes exhibited weak CMG formation defects even in the absence of Mcm10. These findings suggest that the Mcm10-binding region contributes to initial CMG formation.</p>
<p>We also assessed the replication capacity of the mutant Mcm2-7 complexes. For these assays we added the proteins required for CMG formation and initiation of DNA synthesis to a single final incubation (Fig. 3E and 3F). Consistent with the Mcm10-binding and CMG-formation defects observed for Mcm2-7<sup>2-mbm</sup>, DNA synthesis was reduced in reactions containing this mutant complex (Fig. 3E). Although replication with WT Mcm2-7 was fully dependent on Mcm10, the Mcm10 bypass mutants (Mcm2-7<sup>2-bom1</sup> and Mcm2-7<sup>2-bom2</sup>) replicated plasmid DNA in the absence of Mcm10 (Fig. 3F). It was possible that the ability to replicate DNA without Mcm10 was due to co-purification of Mcm10 with Mcm2-7<sup>2-bom1</sup> or Mcm2-7<sup>2-bom2</sup>. In contrast to this possibility, the amount of Mcm10 associated with these complexes was undetectable and lower than the amount required for <em>in vitro</em> DNA replication (Fig. S3). Intriguingly, when Mcm10 was added to reactions containing Mcm2-7<sup>2-bom1</sup> or Mcm2-7<sup>2-bom2</sup>, the resulting replication products were longer than those observed with WT Mcm2-7 (Fig. 3F). This effect on the length of replication products raised the possibility that Mcm10 functions during replication elongation.</p>
<p><strong>Mcm10 stimulates DNA replication elongation</strong></p>
<p>To address the hypothesis that Mcm10 is involved in replication elongation, we titrated the amount of Mcm10 added to the 3-step, reconstituted DNA replication assay (see Fig. 3E) and examined the resulting replication products (Fig. 4A). Consistent with Mcm10 stimulating replication elongation, decreasing amounts of Mcm10 resulted in shorter replication products. Interestingly, the concentrations of Mcm10 that reduce replication-product lengths remain saturating for DNA unwinding and CMG stabilization during initiation (Fig. 4B). This difference in the effective Mcm10 concentration suggests that either the affinity of Mcm10 binding necessary to activate initiation and elongation differs or Mcm10 functions differently during the two events.</p>
<p>The effect of Mcm10 titration on replication-product length was not observed for other helicase-activating proteins. Titrations of Cdc45 or Dpb11 reduced the amount but not the length of the DNA replication products (Fig. S4A and S4B), consistent with an effect on initiation but not elongation. In contrast, titration of the known processivity factor, PCNA <span id="__Fieldmark__530_3205866301" class="anchor"></span>(<span id="__Fieldmark__449_3683559902" class="anchor"></span>P<span id="__Fieldmark__1357_3694128450" class="anchor"></span>relich et al. 1987), showed altered replication-product lengths (Fig. S4C). Because previous studies have suggested that Mcm10 interacts with PCNA <span id="__Fieldmark__541_3205866301" class="anchor"></span>(<span id="__Fieldmark__456_3683559902" class="anchor"></span>D<span id="__Fieldmark__1367_3694128450" class="anchor"></span>as-Bradoo et al. 2006), we asked if the presence of PCNA was required to observe the Mcm10-dependent effects on replication-product length. Although replication products were shorter in the absence of PCNA, reducing Mcm10 levels in this condition further decreased replication product length (Fig. 4C). Thus, Mcm10 impacts replication elongation independent of PCNA.</p>
<p>Because CMG stabilization and replication elongation occurred in the same step in the previous assays, we modified our assay to isolate the effect of Mcm10 on elongation (see Fig. 3C). After CMG assembly, Mcm10 was removed with a high-salt wash. Subsequently, DNA synthesis was activated by addition of DNA polymerases and accessory factors with or without Mcm10. In agreement with an elongation role, addition of Mcm10 to the separate DNA synthesis step resulted in longer DNA replication products (Fig. 4D).</p>
<p><strong>An Mcm10 mutant that is unable to function during elongation</strong></p>
<p>To further understand Mcm10 function, we sought to identify functionally important regions of Mcm10. To this end, we generated <em>MCM10</em> truncations (Fig. 5A) and analyzed their ability to complement the lethal <em>mcm10-FRB</em> anchor-away phenotype (see Fig. 2D). Deletion of the N-terminal domain of Mcm10 resulted in no growth defects. In contrast, several C-terminal domain truncations revealed a region of Mcm10 (residues 399-434) that was critical for viability (Fig. 5B). Alanine scanning of this region identified a mutant (<em>mcm10-A3</em>) that was unable to support cell growth (Fig. 5B).</p>
<p>Given its lethal phenotype, we investigated Mcm10-A3 function <em>in vitro.</em> Like WT Mcm10, Mcm10-A3 co-purified with Mcm2/4/6 and bound to purified Mcm2 and Mcm6 with similar affinity (Fig. S5A and S5B). However, in the context of the CMG complex, Mcm10-A3 showed a ~10-fold reduction in binding affinity (Fig. 6Ai). Despite this binding defect, Mcm10-A3 was comparable to WT Mcm10 in establishing high-salt resistant CMG complexes (Fig. 6Aii and S5C) and stimulating initial DNA unwinding (as measured by RPA recruitment, Fig. 6Ai). In contrast, when incorporated into the complete replication assay, Mcm10-A3 resulted in reduced and shorter replication products compared to WT Mcm10 (Fig. 6Aiii).</p>
<p>To further address whether salt-stable CMG complexes formed with Mcm10-A3 were functional for replication initiation and elongation, we performed replication assays with separate CMG formation and DNA replication steps (see Fig. 3C). CMG complexes were assembled with either WT Mcm10 or Mcm10-A3 followed by a high-salt wash to remove Mcm10 and unstable CMG complexes. In both cases, subsequent addition of WT Mcm10 during the DNA replication elongation step resulted in substantial replication (Fig. 6B). In contrast, addition of Mcm10-A3 during the elongation stage showed background levels of replication independent of whether WT Mcm10 or Mcm10-A3 was present during initial CMG formation. These findings establish that Mcm10-A3 is a separation-of-function mutant that is competent to stabilize the CMG complex and activate initial DNA unwinding but is defective in the stimulation of replication elongation.</p>
<p><strong>Mcm10 stimulates replication elongation <em>in vivo</em></strong></p>
<p>Although our <em>in vitro</em> studies showed that Mcm10 stimulates replication elongation, it was important to determine if Mcm10 contributes to replication elongation <em>in vivo</em>. To this end, hydroxyurea was used to arrest <em>mcm10-1td</em> cells in early S-phase, and Mcm10-1td was degraded by shifting cells to 37°C. At this arrest point, any roles of Mcm10 in CMG formation and initial replisome formation at early-replicating origins have been completed. In addition, <em>CDC7</em> was replaced with <em>cdc7-1</em> to prevent the activation of new origins after release from HU treatment <span id="__Fieldmark__590_3205866301" class="anchor"></span>(<span id="__Fieldmark__501_3683559902" class="anchor"></span>B<span id="__Fieldmark__1666_3694128450" class="anchor"></span>ousset and Diffley 1998; Donaldson et al. 1998). Thus, under non-permissive conditions, only replication elongation by replisomes formed before the HU arrest will determine the rate of completing genome duplication as measured by analysis of DNA content by flow cytometry.</p>
<p>After release from the early-S-phase arrest, comparison of <em>cdc7-1 and cdc7-1</em> <em>mcm10-1td</em> cells revealed that Mcm10-1td degradation resulted in a significant delay in completing S phase (Fig. 6C). Importantly, the elongation defects observed after Mcm10-1td degradation were rescued in cells that expressed <em>MCM10</em> from another locus. Consistent with a defect in elongation stimulation, expression of <em>mcm10-A3</em> failed to rescue the elongation defect of <em>cdc7-1</em> <em>mcm10-1td</em> cells (Fig. 6C). These findings indicate that Mcm10 contributes to replication elongation <em>in vivo</em> and that the stimulation of replication elongation by Mcm10 observed <em>in vitro</em> is not an artifact due to the formation of an incomplete or defective replication forks.</p>
<p><strong>DISCUSSION</strong></p>
<p>Our findings provide multiple insights into the function of Mcm10 during DNA replication. We identified a Mcm10-binding motif at the interface between the OB-fold and A-subdomain of Mcm2 and found that mutants predicted to expose this region restore growth to conditional-lethal alleles of <em>MCM10</em>. We demonstrate that Mcm10 alters the CMG complex in a manner that stabilizes Cdc45 and GINS association with Mcm2-7. Importantly, our data indicate that, in addition to its previously known role during initial helicase activation, Mcm10 stimulates replication elongation. Together, these data support a model in which Mcm10 activates the CMG complex throughout DNA replication.</p>
<p><strong>Mcm10 remodels the CMG complex</strong></p>
<p>We identified a highly-conserved motif in Mcm2 as a binding site for Mcm10. Previous genetic, biochemical and two-hybrid interaction studies support the importance of Mcm10-Mcm2 interactions <span id="__Fieldmark__626_3205866301" class="anchor"></span></p>
<p>(Lee et al. 2010; Quan et al. 2015; Douglas and Diffley 2016; Homesley et al. 2000; Apger et al. 2010)</p>
<p>but had not mapped an Mcm10-binding site. The identified Mcm10-binding motif is buried between the Mcm2 A-subdomain and OB-fold in all current Mcm2-7 structures <span id="__Fieldmark__635_3205866301" class="anchor"></span></p>
<p>(Li et al. 2015; Yuan et al. 2016)</p>
<p>. It is possible that mutants in this motif prevent Mcm10 binding by disrupting a composite Mcm10 binding site that is formed at the interface of the OB-fold and A-subdomain. However, several observations argue against this hypothesis: (1) deletion of the Mcm10-binding motif inhibits Mcm10 binding in the absence of the OB-fold (Fig. 1D, lanes 15 and 19); (2) Mcm10 does not bind the OB-fold alone (Fig. 1D, lane 16); and (3) a protein fragment including the OB-fold and the A-subdomain does not bind Mcm10 better than the A-subdomain alone (Fig. 1D, lanes 12 and 15).</p>
<p>Instead of binding to a site formed by both the OB-fold and A-subdomain, we propose that Mcm10 induces or captures a conformational change in Mcm2 that exposes the Mcm10-binding motif resulting in CMG activation. Consistent with this hypothesis mutations on both sides of the Mcm2 OB-fold/A-subdomain interface designed to expose the Mcm10-binding motif restore viability to cells with conditional-lethal <em>MCM10</em> alleles (Fig. 2 and S1). In addition, Mcm2-7 complexes containing these mutations allow replication initiation in the absence of Mcm10 <em>in vitro</em> (Fig. 3F). Although the Mcm10-binding motif is buried in current <em>S. cerevisiae</em> Mcm2-7 structures <span id="__Fieldmark__650_3205866301" class="anchor"></span></p>
<p>(Yuan et al. 2016; Li et al. 2015)</p>
<p>, the A-subdomain is rotated and the Mcm10-binding motif exposed in the only full-length structure of an active archaeal MCM complex <span id="__Fieldmark__658_3205866301" class="anchor"></span></p>
<p>(Miller et al. 2014)</p>
<p>; Fig. S6). We note that this archaeal MCM complex is a hybrid protein with the N-terminal domain (including both the A-subdomain and the OB-fold) from <em>S. sulfolobus</em> and the C-terminal AAA+ domain from <em>P. furiosus</em>. Nevertheless, this hybrid MCM is an active helicase and there are no unusual interactions between the N- and C-terminal domains that would drive movement of the A-subdomain.</p>
<p>Further evidence in favor of Mcm10 altering CMG conformation stems from our observation that Mcm10 stabilizes and activates the CMG complex (Fig. 3 and 6). Consistent with the Mcm10-dependent CMG stabilization being due to a conformational change, we find that stabilization does not require the continued presence of Mcm10 (Fig. 3B). It is unclear what molecular event causes CMG stabilization and when it occurs relative to helicase activation. Mcm10-dependent movement of the Mcm2 A-subdomain could reveal additional interaction regions on Mcm2-7 for Cdc45 and GINS, resulting in enhanced stability and helicase activation. Alternatively, Mcm10-dependent stabilization of the CMG complex could occur as a consequence of helicase activation or extrusion of ssDNA from the Mcm2-7 central channel <span id="__Fieldmark__671_3205866301" class="anchor"></span>(<span id="__Fieldmark__578_3683559902" class="anchor"></span>F<span id="__Fieldmark__1968_3694128450" class="anchor"></span>u et al. 2011). For example, the ssDNA generated by one or both of these events could interact with Cdc45 or GINS <span id="__Fieldmark__682_3205866301" class="anchor"></span>(<span id="__Fieldmark__585_3683559902" class="anchor"></span>C<span id="__Fieldmark__1973_3694128450" class="anchor"></span>osta et al. 2014) resulting in stabilized CMG complexes. Supporting this possibility, Cdc45 is related to the bacterial RecJ ssDNA nuclease and has been shown to bind ssDNA <span id="__Fieldmark__693_3205866301" class="anchor"></span>(<span id="__Fieldmark__592_3683559902" class="anchor"></span>B<span id="__Fieldmark__1978_3694128450" class="anchor"></span>ruck and Kaplan 2013; Petojevic et al. 2015). Finally, given the potential role of OB-fold domains in ssDNA interactions <span id="__Fieldmark__705_3205866301" class="anchor"></span>(<span id="__Fieldmark__600_3683559902" class="anchor"></span>F<span id="__Fieldmark__1984_3694128450" class="anchor"></span>roelich et al. 2014; Ashton et al. 2013) it is also possible that release from the A-subdomain allows the Mcm2 OB-fold domain to form more productive interactions with translocating ssDNA. These possibilities are not mutually exclusive.</p>
<p>Our studies combined with previous data suggest that the Mcm2 A-domain/OB-fold interface is a nexus for interactions that regulate Mcm2-7 activity. In addition to inhibiting Mcm10 binding, mutations at this interface also lead to reduced Cdc45 and GINS recruitment (Fig. 3D). These defects are consistent with interactions between Cdc45 and Mcm2 A-subdomain observed in the CMG structure (Yuan et al., 2015, Fig. 2A). Interestingly, of the three OB-fold/A-subdomain interface mutants we tested <em>in vitro</em>, the stronger Mcm10 bypass allele (<em>mcm2-bom1</em>) has only minor CMG formation defects (Fig. 3D). Thus, bypassing Mcm10 function may involve a balance between opening the OB-fold/A-subdomain interface while not disrupting interactions necessary for Cdc45 and GINS binding.</p>
<p>Several explanations are possible for <em>mcm2-bom1</em> and <em>mcm2-bom1</em> not being able to bypass a complete <em>MCM10</em> deletion (Fig. S1C). It is possible that a small amount of residual Mcm10 function is required to allow cells to grow in the presence of the bypass alleles. Futhermore, the inability to bypass <em>Δmcm10</em> could be due to incomplete disruption of the A-subdomain/OB-fold interaction in <em>mcm2-bom1</em> or <em>mcm2-bom2</em>. Given that Mcm10 catalyzes the committed step of replication initiation, another possibility is that Mcm10 bypass may lead to a deleterious loss of coordination between replication initiation events. Alternatively, Mcm10 could have an additional essential function beyond helicase activation.</p>
<p><strong>Mcm10 stimulates replication elongation</strong></p>
<p>We provide both <em>in vivo</em> (Fig. 6) and <em>in vitro</em> (Fig. 4) evidence that Mcm10 stimulates replication elongation. Consistent with a role for Mcm10 in elongation, previous studies have found that Mcm10 travels with the replisome <span id="__Fieldmark__745_3205866301" class="anchor"></span>(<span id="__Fieldmark__636_3683559902" class="anchor"></span>G<span id="__Fieldmark__2051_3694128450" class="anchor"></span>ambus et al. 2006; Pacek et al. 2006; Ricke and Bielinsky 2004). <span id="__Fieldmark__756_3205866301" class="anchor"></span>(<span id="__Fieldmark__643_3683559902" class="anchor"></span>G<span id="__Fieldmark__2056_3694128450" class="anchor"></span>ambus et al., 2006; Pacek et al., 2006). Furthermore, a temperature-sensitive allele of <em>MCM10</em> (<em>mcm10-1</em>) causes replication fork pausing at the restricted temperature <span id="__Fieldmark__773_3205866301" class="anchor"></span>(<span id="__Fieldmark__656_3683559902" class="anchor"></span>H<span id="__Fieldmark__2070_3694128450" class="anchor"></span>omesley et al. 2000; Merchant et al. 1997). Supporting the importance of this function, we note that the elongation-defective <em>mcm10-A3</em> allele is unable to complement the lethal depletion of Mcm10-FRB (Fig. 6B).</p>
<p>Although a precise mechanism for Mcm10 stimulation of elongation remains to be determined, our studies provide insights into this control. The finding that Mcm10 stabilizes the CMG complex (Fig. 3B) raises the possibility that Mcm10 binding stimulates elongation by enhancing the processivity of the CMG complex. In addition, Mcm2-7<sup>2-bom1</sup> and Mcm2-7<sup>2-bom2</sup> both lead to longer replication products suggesting that conformational changes in the OB-fold/A-subdomain interface contribute to elongation. It is possible that Mcm10 binding drives changes in the OB-fold/A-subdomain interface and that this has a direct impact on the stability or speed of the CMG. Alternatively, changes induced by Mcm10 binding could alter interactions of Cdc45 and GINS with Mcm2-7. Further detailed biochemical studies will be required to test these possibilities.</p>
<p><strong>Does Mcm10 activate initiation and elongation by the same mechanism?</strong></p>
<p>Whether Mcm10 functions during replication initiation and elongation by the same or different mechanisms remains to be determined. The simplest model is that Mcm10 stimulates both events by the same mechanism. Consistent with this idea, our <em>in vitro</em> analyses of the Mcm10-bypass mutants suggest that both the initiation and elongation functions of Mcm10 are impacted by these mutants. The ability to detect replication products in these assays indicates that these mutants facilitate initiation in the absence of Mcm10 (Fig. 3F). Two observations suggest that the elongation function of Mcm10 is also altered by these mutations. First, in the absence of Mcm10, the length of <em>in vitro</em> replication products correlates with the strength of the Mcm10 bypass allele (Fig. 3F). Second, when WT Mcm10 is present, Mcm2-7<sup>2-bom1</sup> and Mcm2-7<sup>2-bom2</sup> produce longer replication products (Fig. 3F).</p>
<p>On the other hand, we identified an <em>MCM10</em> allele (<em>mcm10-A3</em>) that shows differential effects on replication initiation and elongation. This protein is defective for stimulation of replication elongation (Fig. 6Aiii) and binding to the CMG (Fig. 6Ai), but exhibits similar capabilities as WT Mcm10 to form salt-stable CMG complexes and stimulate initial DNA unwinding (Figs. 6Ai, 6Aii and S5C). These findings suggest that stable binding to the CMG correlates with the ability to stimulate replication elongation and suggests that a different interaction is involved in stabilizing the CMG and stimulating initial DNA unwinding. Further experiments will be necessary to determine if and how the Mcm10 mechanism of function differs between initiation and elongation.</p>
<p><strong>MATERIALS AND METHODS</strong></p>
<p><strong>Yeast strains and plasmids</strong></p>
<p>All <em>S. cerevisiae</em> strains were congenic with W303 (<em>ade2-1 trp1-1 leu2-3,112 his3-11,15 ura3-1 can1-100</em>) and the genotypes are summarized in table S2. Protein expression plasmids are summarized in table S3.</p>
<p><strong>Protein Purification</strong></p>
<p>Mcm2-7/Cdt1, ORC, Cdc6, Ctf4 and Top2 were purified as previously described <span id="__Fieldmark__817_3205866301" class="anchor"></span>(<span id="__Fieldmark__696_3683559902" class="anchor"></span>Y<span id="__Fieldmark__2199_3694128450" class="anchor"></span>eeles et al. 2015; Kang et al. 2014). Purifications of the remaining proteins are described below.</p>
<p><strong>Buffers</strong></p>
<p>The following buffers were used for protein purification: buffer H (50 mM HEPES-KOH [pH 7.6], 1 mM EDTA, 1 mM EGTA, 5 mM MgOAc, 10% glycerol), buffer I (buffer H, 0.02% NP-40, 0.3 M potassium glutamate [KGlut], 10 mM imidazole), buffer M (buffer H, 0.02% NP-40, 0.3 M KCl), buffer D (buffer H, 0.3 M KOAc, 0.02% NP-40), buffer E (buffer H, 0.4 M NaOAc, 0.01% NP-40), buffer R (50 mM HEPES-KOH [pH 7.6], 10% glycerol, 7 mM MgOAc, 0.01% NP-40, 1 mM ATP) and buffer C (25 mM Tris-Cl [pH 7.2], 10% glycerol, 1 mM DTT).</p>
<p><strong>Yeast cell growth and lysis</strong></p>
<p>All yeast strains were grown in selective media before being inoculated into 8L of YEP + 2% glycerol at 30°C. Cell were grown to an OD<sub>600</sub> ~1 before induction with galactose (2% final conc.). After 4-6 hours, the cells were harvested and washed with 200 ml of chilled water + 0.2 mM PMSF. The cells were then resuspended in approximately 1/2 packed cell volume of the indicated lysis buffer containing a protease inhibitor tablet and frozen dropwise into liquid nitrogen. The frozen cells were lysed using a SPEX SamplePrep Freezer/Mill. Lysed cell powder was transferred to ultracentrifugation tubes and thawed on ice. The lysate was cleared by centrifugation in a Beckman ultracentrifuge at ≥140k x g for ≥1 hour. All steps were done at 4°C.</p>
<p><strong>FLAG-affinity purification</strong></p>
<p>Cleared lysates were incubated with the indicated amount of packed anti-FLAG M2 affinity gel (Sigma) for 2 hours at 4°C. After a column wash, the bound proteins were eluted with the indicated buffer including 0.2 mg/ml 3xFLAG peptide (MDYKDHDGDYKDHDIDYKDDDDK, Koch Institute Swanson Biotechnology Center). The first eluate was collected by flowing over 1 column volume (CV) of elution buffer. The next four eluates were collected after a 30-minute incubation with the elution buffer.</p>
<p><strong>S-CDK</strong></p>
<p>Clb5-FLAG and Cdc28-6xHis were over-expressed from ySK119. Clb5 was expressed with a deletion of residues 1-94 to remove a destruction box <span id="__Fieldmark__840_3205866301" class="anchor"></span>(<span id="__Fieldmark__715_3683559902" class="anchor"></span>C<span id="__Fieldmark__2220_3694128450" class="anchor"></span>ross et al. 1999). Cells were resuspended in buffer H, 1 M sorbitol, 0.02% NP-40, 2 mM ATP, 0.5 M KCl. After cell lysis, the cleared lysate was diluted to 0.3 M KCl with buffer H. The lysate was then incubated with 1 ml anti-FLAG M2 affinity gel equilibrated with buffer M. The resin was washed with 20 CV of buffer M followed by 10 CV of buffer I + 3xFLAG peptide. S-CDK was eluted in buffer I. S-CDK-containing fractions were flowed over cOmplete His-tag resin (Roche) twice, washed with 20 CV of buffer I, and eluted with buffer I + 250 mM imidazole. Peak fractions were pooled and applied to a Superdex 200 column (GE healthcare) equilibrated with buffer H, 0.01 NP-40, 1 mM ATP, 0.3 M KGlut.</p>
<p><strong>Sld3/Sld7</strong></p>
<p>Sld3-3xFLAG and Sld7-VSV-G were over-expressed from ySK123. Sld3 was expressed with a deletion of residues 1-104 to remove a putative destruction box. Cells were resuspended in buffer H, 1 M sorbitol, 0.02% NP-40, 2 mM ATP, 0.8 M KCl. After cell lysis, the cleared lysate was diluted to 0.3 M KCl with buffer H. The diluted lysate was incubated with 1.5 ml anti-FLAG M2 affinity gel equilibrated with buffer M. The resin was washed with 30 CV of buffer M and eluted in buffer M + 3xFLAG peptide. Sld3/Sld7 containing fractions were diluted to to 0.2 M KCl with buffer H immediately before being applied to a 1 ml HiTrap SP HP column (GE healthcare). The column was washed with buffer H, 0.02% NP-40, 330 mM KCl and eluted with buffer H, 0.02% NP-40, 640 mM KCl.</p>
<p><strong>Sld2</strong></p>
<p>3xFLAG-3C-Sld2 was over-expressed from ySK127. Cells were resuspended in buffer H, 1 M sorbitol, 0.02% NP-40, 2 mM ATP, 0.8 M KCl. After cell lysis, the cleared lysate was dialyzed overnight (16 hours) in buffer M with 3 mM ATP and 1 mM PMSF. The lysate was cleared a second time by spinning for 15 min at 11k rpm. Sld2 was purified using 1 ml anti-FLAG resin as described above for Sld3/Sld7 except 1 mM ATP was added to buffer M. Sld2 containing fractions were diluted to 0.2 M KCl with buffer H immediately before being applied to a 1 ml HiTrap SP HP column. Sld2 was eluted with a 15 CV gradient of 0.2 M – 1 M KCl in buffer H, 0.02% NP-40, 1 mM ATP.</p>
<p><strong>Dpb11</strong></p>
<p>Dpb11-FLAG was over-expressed from yRH154. Dpb11 was purified in a similar manner as Sld2 except for the following modifications. Fractions containing Dpb11 from the anti-FLAG column were diluted to 0.1 M KCl with buffer H immediately before being applied to a 1 ml HiTrap SP HP column. Dpb11 was eluted with an 18 CV gradient of 0.1 M – 1 M KCl in buffer H, 0.02% NP-40, 1 mM ATP. The peak fractions were dialyzed against buffer D.</p>
<p><strong>Cdc45</strong></p>
<p>Cdc45-3xFLAG was over-expressed from yMM016. Purification of Cdc45 was based on a previously published protocol (Yeeles et al., 2015) with the following modifications. Cells were resuspended in buffer H, 1 M sorbitol, 3 mM ATP, 500 mM KGlut. After lysis, the lysate was incubated with 1.5 ml anti-FLAG M2 affinity gel equilibrated with buffer H, 500 mM KGlut, 2 mM ATP for. The resin was washed with 20 CV of buffer H, 500 mM KGlut, 2 mM ATP followed by 10 CV of 20 mM potassium phosphate buffer (pH 7.4), 150 mM KOAc, 10% glycerol. Cdc45 was eluted in the previous buffer + 3xFLAG peptide. After the hydroxyapatite column, Cdc45 was dialyzed against buffer H, 0.3 M KGlut.</p>
<p><strong>GINS</strong></p>
<p>Sld5, Psf1, Psf3, and Psf2-3C-6xHis-FLAG were over-expressed from ySK136. Cells were resuspended in buffer H, 1 M sorbitol, 0.02% NP-40, 2 mM ATP, 0.5 M KCl. After lysis, the cleared lysate was diluted to 0.3 M KCl with buffer H. The lysate was then incubated with 1.5 ml anti-FLAG M2 affinity gel equilibrated with buffer M. The resin was washed with 20 CV of buffer M followed by 10 CV of buffer H, 0.02% NP-40, 0.1 M KCl. GINS was eluted in the previous buffer + 3xFLAG peptide. The FLAG tag on Psf2 was removed with an overnight incubation (16 hours) with HRV 3C protease. GINS was flowed over cOmplete His-tag resin to remove uncut GINS and HRV 3C protease before applying the flow through to a 1 ml HiTrap Q HP column (GE healthcare). GINS was eluted with a 20 CV gradient of 0.1 M – 1 M KCl in buffer H, 0.02% NP-40. The peak fractions were dialyzed against buffer D.</p>
<p><strong>Polymerase ε</strong></p>
<p>Pol2-3C-5xFLAG, Dpb3, Dpb4-3C-6xHis, and Dpb2-3C-FLAG were over-expressed from yMH28. Cells were resuspended in buffer E. After cell lysis, the cleared lysate was incubated with 1.5 ml anti-FLAG M2 affinity gel equilibrated with buffer E. The resin was washed with 20 CV of buffer E and eluted in buffer E + 3xFLAG peptide. The FLAG tags were removed with a 2-hour incubation with HRV 3C protease. Polymerase <strong>ε</strong> was concentrated using a 10K MWCO spin column (Sartorius) before being applied to a Superdex 200 column equilibrated with buffer E.</p>
<p><strong>Polymerase α/primase</strong></p>
<p>Pri1, Pri2, Pol1, and Pol12-3C-FLAG were over-expressed from yAS3. Polymerase α/primase was purified in a similar manner as polymerase <strong>ε</strong> except buffer H, 0.3 M KGlut, 0.01% NP-40 was used for cell resuspension and all chromatography steps and an additional cOmplete protease inhibitor tablet was added during the anti-FLAG incubation.</p>
<p><strong>DDK</strong></p>
<p>Dbf4-FLAG and Cdc7 were over-expressed from yRH146. Cells were resuspended in buffer H, 0.3 M KGlut, 0.01% NP-40. After lysis, the cleared lysate was incubated with 1.5 ml anti-FLAG M2 affinity gel equilibrated with buffer H, 0.3 M KGlut, 0.01% NP-40. The resin was washed with 30 CV and eluted in the previous buffer + 3xFLAG peptide.</p>
<p><strong>RPA</strong></p>
<p>The purification was based on a previously published protocol <span id="__Fieldmark__880_3205866301" class="anchor"></span>(<span id="__Fieldmark__751_3683559902" class="anchor"></span>G<span id="__Fieldmark__2256_3694128450" class="anchor"></span>ibb et al. 2014) with the following modifications. Rosetta 2 <em>E. coli</em> cells were transformed with p11d-tscRPA-30MxeHis6 and 2 L of culture were grown at 37°C in 2xYT+amp+cm. After the Ni-NTA and chitin column, RPA was applied to a Superdex 200 column equilibrated with 20 mM Tris-HCl (pH 8.0), 150 mM NaCl, 1 mM EDTA, 20% glycerol.</p>
<p><strong>Mcm10 and Mcm10-A3</strong></p>
<p>WT Mcm10-FLAG and Mcm10-A3-FLAG were over-expressed from MLy049 and MLy136, respectively. FLAG-3C-Mcm10-V5 was over-expressed from MLy048. Cells were resuspended in buffer H, 1 M sorbitol, 0.05% NP-40, 0.5 M KCl. After lysis, the cleared lysate was diluted to 0.25 M KCl with buffer H, 0.05% NP-40. The lysate was then incubated with 1.5 ml anti-FLAG M2 affinity gel equilibrated with buffer H, 0.05% NP-40, 0.25 M KCl. The resin was washed with 30 CV of buffer H, 0.05% NP-40, 0.25 M KCl and eluted in the same buffer. The eluted protein was diluted to 0.15 M KCl with buffer H, 0.05% NP-40 immediately before being applied to a 1 ml HiTrap SP HP column. Mcm10 was eluted with a 15 CV gradient of 0.15 M – 1.5 M KCl in buffer H, 0.05% NP-40.</p>
<p><strong>Polymerase δ</strong></p>
<p>Purification of Polymerase δ was based on a previously published protocol <span id="__Fieldmark__906_3205866301" class="anchor"></span>(<span id="__Fieldmark__773_3683559902" class="anchor"></span>L<span id="__Fieldmark__2276_3694128450" class="anchor"></span>angston and O'Donnell 2008) with the following modifications. 8 L of both yeast and <em>E. coli</em> cultures were used. Pol3-FLAG was over-expressed from yAS26. Rosetta 2 <em>E. coli</em> cells were co-transformed with pMM051 and pMM053 and grown at 37°C in 2xYT media with 100 μg/ml ampicillin, 50 μg/ml kanamycin, and 34 μg/ml chloramphenicol. (2xYT+amp+kan+cm). At an OD<sub>600</sub> ~0.6, cells were moved to 25°C and induced with 1 mM IPTG. After a 4-hour induction, cells were harvested and resuspended in 15 mL (per 1L culture) of buffer H, 0.01% NP-40, 0.3 M KCl. The resuspended <em>E. coli</em> cells were treated with 0.1 mg/ml lysozyme (from chicken egg white, Sigma) for 30 minutes followed by 6 cycles of sonication (30% amplitude, 10 seconds ON, 10 seconds OFF) with a Branson digital sonifier (Emerson Industrial Automation). Yeast cells were resuspened in buffer H, 0.01% NP-40, 0.3 M KCl. After lysis and clarification, yeast and <em>E. coli</em> lysates were combined and Pol δ was purified in a similar manner as Dpb11.</p>
<p><strong>RFC</strong></p>
<p>Purification of RFC with a deletion of RFC1 from residues 1-274 was based on a previously published protocol <span id="__Fieldmark__934_3205866301" class="anchor"></span>(<span id="__Fieldmark__797_3683559902" class="anchor"></span>G<span id="__Fieldmark__2301_3694128450" class="anchor"></span>omes et al. 2000) with the following modifications. Rosetta 2 <em>E. coli</em> cells were transformed with pBL481 and 4 L of culture were grown at 37°C in 2xYT+amp+cm. At an OD<sub>600</sub> ~0.7, cells were moved to 30°C and induced with 0.5 mM IPTG. After a 3-hour induction, cells were harvested in 20 mL (per 1L culture) of buffer R, 0.2 M NaCl plus a cOmplete protease inhibitor tablet. The resuspended cells were treated with lysozyme and sonicated as described for Pol δ. The cleared lysate was applied to 2 ml Ni-NTA resin equilibrated with buffer R, 0.2 M NaCl. The resin was washed with 50 CV of buffer R, 0.2 M NaCl and eluted with the same buffer with 300 mM imidazole. The eluted protein was diluted to 0.15 M NaCl with buffer R before being applied to a 1 mL HiTrap SP HP column. RFC was eluted with a 24 CV gradient from 0.15 M – 0.75 M NaCl in buffer R. Peak fractions were pooled and applied to a Superdex 200 column equilibrated with buffer R, 0.15 M NaCl.</p>
<p><strong>PCNA</strong></p>
<p>Rosetta 2 <em>E. coli</em> cells were transformed with pMM054 and 1 L of culture was grown at 37°C in 2xYT+amp+cm. At an OD<sub>600</sub> ~0.6, cells were induced with 1 mM IPTG. After 3 hours, cells were harvested in 20 mL of 50 mM HEPES-KOH (pH 7.6), 0.1 M KCl, 10% glycerol, 0.01% NP-40, 10 mM imidazole plus a protease inhibitor tablet. The resuspended cells were treated with lysozyme and sonicated as described for Pol δ.. The cleared lysate was applied to 3 ml Ni-NTA resin equilibrated with the previous buffer. The resin was washed and then eluted with 300 mM imidazole in the same buffer. PCNA containing fractions were applied to a 1 mL HiTrap SP HP column and eluted with a 20 CV gradient from 0.1 M – 1 M KCl in buffer H.</p>
<p><strong>Mcm2, Mcm4, Mcm5, Mcm6, Mcm7 and Mcm2 truncations</strong></p>
<p>Mcm2 (pNI001), Mcm4 (pNI002), Mcm5 (pNI003), Mcm6 (pNI004), Mcm7 (pNI005), Mcm2 1-483 (pML028), Mcm2 1-195 (pML027) were C-terminally 6xHis tagged and expressed in Rosetta 2 <em>E. coli</em> cells. Resuspension of the bacterial pellet and purification was done in buffer H (without EDTA and EGTA), 0.25 M KCl, 10 mM Imidazole. Mcm2 474-868 (pML035), Mcm2 196-299 (pML030), Mcm2 300-473 (pML034), Mcm2 196-240 (pML032), Mcm2 241-299 (pML033), Mcm2 196-289 (pML031) and Mcm2 196-473-mbm (pML050) were N-terminally MBP tagged and expressed in Rosetta 2 <em>E. coli</em> cells<em>.</em> Resuspension of the bacterial pellet and purification was done in buffer H, 250 mM KCl. For both 6xHis and MBP purifications cells were resuspended in 50 ml of buffer and treated with lysozyme and sonicated as described for Pol δ. The cell lysate was cleared by a 30-minute centrifugation (20,000 g). 2 ml of Ni-NTA resin was used for 6xHis purifications and 2 ml of amylose resin was used for MBP purifications. Before elusion, resin was washed with 30 ml of buffer and proteins were eluded with 350 mM imidazole from the Ni-NTA resin and with 10 mM maltose from the amylose resin.</p>
<p><strong>Reconstituted DNA replication and CMG formation</strong></p>
<p>The DNA plasmid template, pUC19-<em>ARS1</em>, was randomly biotinylated and coupled to streptavidin-coated magnetic beads as previously described <span id="__Fieldmark__982_3205866301" class="anchor"></span>(<span id="__Fieldmark__841_3683559902" class="anchor"></span>H<span id="__Fieldmark__2368_3694128450" class="anchor"></span>eller et al. 2011). Each incubation step was performed in a thermomixer (Eppendorf) shaking at 1150 rpm at 25<sup>o</sup>C. Supernatants of each step were removed by applying the reaction to a DynaMag-2 Magnet (ThermoFisher Scientific) to isolate the DNA coupled to magnetic streptavidin beads from the supernatant. Mcm2-7 loading was performed by incubating 0.25 pmol ORC, 0.5 pmol Cdc45, and 1 pmol Mcm2-7/Cdt1 with 0.125 pmol pUC19-<em>ARS1</em> in 25 mM HEPES (pH 7.6), 10 mM MgOAc, 0.1 mM ZnOAc, 1 mM DTT, 300 mM KGlut, 20 mM phosphocreatine (PC), 6 mM ATP, 0.1 mM EDTA, 0.02% NP-40, 10% glycerol and 0.2 μg creatine kinase (CK). The Mcm2-7 loading step was done in a volume of 10 μl and was incubated for 30 minutes. After removal of the supernatant, DDK phosphorylation was performed as described previously <span id="__Fieldmark__997_3205866301" class="anchor"></span>(<span id="__Fieldmark__852_3683559902" class="anchor"></span>K<span id="__Fieldmark__2377_3694128450" class="anchor"></span>ang et al. 2014) in a 10 μl reaction volume for 25 minutes. After removal of the DDK reaction supernatant, the following amounts of protein were added to the DDK-phosphorylated Mcm2-7: 1 pmol Cdc28/Clb5, 0.3 pmol DDK, 1 pmol Sld3/Sld7, 5 pmol Cdc45, 2 pmol Sld2, 0.6 pmol Dpb11, 5 pmol GINS, 0.15 pmol Mcm10 (or as indicated), 1.85 pmol Pol <strong>ε</strong>, and 2 pmol RPA. The buffer used for CMG formation contained 25 mM HEPES, 12 mM MgOAc, 0.1 mM ZnOAc, 1 mM DTT, 20 mM PC, 6 mM ATP, 10% glycerol, 0.04 mg/ml BSA and 0.3 μg CK. The CMG formation step was done in a volume of 30 μl and was incubated for 1 hour. Reactions were washed with the indicated buffer, and proteins were released from the DNA by incubating with 5 U DNase (Worthington) in 15 μl buffer H, 150 mM KGlut, 0.01% NP-40 for 30 minutes at 25<sup>o</sup>C before immunoblotting.</p>
<p>To initiate DNA replication, the following amounts of proteins were added along with the proteins from the CMG formation step: 2.5 pmol Pol α, 0.5 pmol Top2, 3 pmol Ctf4, 1 pmol RFC, 6 pmol PCNA, 2 pmol Pol <strong>δ</strong>. 0.2 mM rNTP, 0.04 mM dNTP, and 10 μCi [α-P<sup>32</sup>] dCTP were included in the buffer to initiate and monitor DNA replication. The DNA replication step was done in a volume of 30 μl and was incubated for 1 hour. Reactions were washed with buffer H, 500 mM NaCl, 0.05% NP-40 before being resuspended in alkaline gel-loading buffer (50 mM NaOH, 4 mM EDTA, 4.5% Ficoll400, 0.01% bromocresol green). DNA replication products were separated in a 1% alkaline agarose gel, dried and imaged using a phosphor screen. When CMG formation and DNA replication were performed in separate steps, the supernatant from the CMG formation step was removed after 1 hour before adding the DNA replication proteins. DNA replication was initiated by omitting all the proteins used for CMG formation except Pol <strong>ε</strong>, RPA, and Mcm10 (as indicated). The CMG formation step and DNA replication step were both done in a volume of 30 μl and were incubated for 1 hour each.</p>
<p>The following antibodies were used for immunoblotting: α-Cdc45 (HM7135), α-GINS (HM7128), α-Mcm10 (HM6465), α-Pol <strong>ε</strong> (HM7602), α-Mcm2 (yN-19, Santa Cruz), α-Mcm5 (yN-19, Santa Cruz), and α-Rfa1 (gift from Steven Brill).</p>
<p><strong>Anchor-away</strong></p>
<p>The base strain (MLy054) for protein anchoring was obtained by crossing Y40434 (Euroscarf; Haruki et al., 2008) and OAy470 to obtain a <em>bar1::hisG MATa</em> version of the Y40434 strain. Next MCM10 or CDC7 were C-terminally FRB tagged using plasmids pFA6a-FRB-GFP-KanMX6 or pFA6a-FRB-GFP-His3 (Euroscarf), respectively. All alleles for <em>mcm10-FRB</em> complementation were expressed from the <em>MCM10</em> promoter and were inserted into <em>LEU2</em> locus as a single-copy integration. To drive protein anchoring, solid media was supplemented with DMSO (1% final conc.) and rapamycin (5 μg/ml final conc.), whereas liquid media was supplemented with DMSO (4% final conc.) and rapamycin (100 μg/ml final conc.).</p>
<p><strong>Mass Spectrometry</strong></p>
<p>An eluate from anti-FLAG M2 affinity gel (Sigma) of an Mcm10-FLAG purification was separated on an SDS-PAGE gel, and the band corresponding to Mcm10-FLAG was excised. The remainder of the gel lane was subjected to mass spectrometry using standard methods.</p>
<p><strong>Immunoprecipitation (IP)</strong></p>
<p>Purified Mcm10-FLAG was bound to anti-FLAG M2 affinity gel in buffer H, 250 mM KCl, 0.05% NP-40 for 1 hour at 4<sup>o</sup>C. Purified Mcm2, Mcm4, Mcm5, Mcm6, Mcm7, or variants of Mcm2 were added to bound Mcm10, incubated at 25<sup>o</sup>C for 30 minutes and washed 3 times in buffer H, 250 mM KCl, 0.05% NP-40 (unless stated otherwise in the figure legend). Precipitated proteins were eluded in buffer H, 250 mM KCl, 0.05% NP-40 with 0.15 mg/ml 3xFLAG peptide.</p>
<p><strong>Flow cytometry</strong></p>
<p>Cells were arrested in G1 phase with 20 μg/ml α-factor on the hour and arrested in early S phase with 150 mM HU. Cells were released from HU arrest into media containing 1.5 μg/ml nocodozole. For each time point, 0.5 ml of cells (OD<sub>600</sub> ~0.6) were fixed in 10 ml of 70% ethanol for at least 15 minutes. The cells were then washed once with 1ml of 50 mM sodium citrate. RNA was degraded with 10 μg/ml RNase A in 500 μl of 50 mM sodium citrate for 16 hours at 37°C, followed by 30 minutes 20 μg/ml Proteinase K treatment at 42°C. DNA was stained with 10×SYTOX Green in 100 μl of 50 mM sodium citrate for 30 minutes and analyzed with CytoFLEX Flow Cytometer (Beckman Coulter).</p>
<p><strong>ACKNOWLEDGEMENTS</strong></p>
<p>We thank Megan Warner, David Phizicky, Terry Orr-Weaver and Bob Sauer for helpful comments on the manuscript, Adam Shoemaker, Nikola Ivica, Margaret Himmelright and Sukhyun Kang for protein expression strains, Eric C. Greene, Peter M.J. Burgers and Steven J. Brill protein expression plasmids and antibodies and members of the Bell laboratory for useful discussions. M.L. was supported by a postdoctoral fellowship from the Leukemia &amp; Lymphoma Society (5336-15). M.F.M. was supported by a graduate fellowship from the David H. Koch Fellowship Fund. S.P.B. is an investigator with the Howard Hughes Medical Institute. This work was supported in part by the Koch Institute Support Grant P30-CA14051 from the NCI. We thank the Koch Institute Swanson Biotechnology Center for technical support, specifically the Biopolymers core.</p>
<p><strong>REFERENCES</strong></p>
<p><span id="__Fieldmark__1073_3205866301" class="anchor"></span>A<span id="__Fieldmark__924_3683559902" class="anchor"></span>b<span id="__Fieldmark__2517_3694128450" class="anchor"></span>id Ali F, Renault L, Gannon J, Gahlon HL, Kotecha A, Zhou JC, Rueda D, Costa A. 2016. Cryo-EM structures of the eukaryotic replicative helicase bound to a translocation substrate. <em>Nat Commun</em> <strong>7</strong>: 10708.</p>
<p>Apger J, Reubens M, Henderson L, Gouge CA, Ilic N, Zhou HH, Christensen TW. 2010. Multiple functions for Drosophila Mcm10 suggested through analysis of two Mcm10 mutant alleles. <em>Genetics</em> <strong>185</strong>: 1151–1165.</p>
<p>Ashton NW, Bolderson E, Cubeddu L, O'Byrne KJ, Richard DJ. 2013. Human single-stranded DNA binding proteins are essential for maintaining genomic stability. <em>BMC Mol Biol</em> <strong>14</strong>: 9.</p>
<p>Bousset K, Diffley JF. 1998. The Cdc7 protein kinase is required for origin firing during S phase. <em>Genes Dev</em> <strong>12</strong>: 480–490.</p>
<p>Bruck I, Kaplan DL. 2013. Cdc45 protein-single-stranded DNA interaction is important for stalling the helicase during replication stress. <em>J Biol Chem</em> <strong>288</strong>: 7550–7563.</p>
<p>Costa A, Renault L, Swuec P, Petojevic T, Pesavento JJ, Ilves I, MacLellan-Gibson K, Fleck RA, Botchan MR, Berger JM. 2014. DNA binding polarity, dimerization, and ATPase ring remodeling in the CMG helicase of the eukaryotic replisome. <em>eLife</em> <strong>3</strong>: e03273.</p>
<p>Cross FR, Yuste-Rojas M, Gray S, Jacobson MD. 1999. Specialization and targeting of B-type cyclins. <em>Mol Cell</em> <strong>4</strong>: 11–19.</p>
<p>Das-Bradoo S, Ricke RM, Bielinsky AK. 2006. Interaction between PCNA and Diubiquitinated Mcm10 Is Essential for Cell Growth in Budding Yeast. <em>Mol Cell Biol</em> <strong>26</strong>: 4806–4817.</p>
<p>Deegan TD, Yeeles JT, Diffley JF. 2016. Phosphopeptide binding by Sld3 links Dbf4-dependent kinase to MCM replicative helicase activation. <em>EMBO J</em> <strong>35</strong>: 961–973.</p>
<p>Donaldson AD, Fangman WL, Brewer BJ. 1998. Cdc7 is required throughout the yeast S phase to activate replication origins. <em>Genes Dev</em> <strong>12</strong>: 491–501.</p>
<p>Douglas ME, Diffley JFX. 2016. Recruitment of Mcm10 to Sites of Replication Initiation Requires Direct Binding to the Minichromosome Maintenance (MCM) Complex. <em>J Biol Chem</em> <strong>291</strong>: 5879–5888.</p>
<p>Evrin C, Clarke P, Zech J, Lurz R, Sun J, Uhle S, Li H, Stillman B, Speck C. 2009. A double-hexameric MCM2-7 complex is loaded onto origin DNA during licensing of eukaryotic DNA replication. <em>Proc Natl Acad Sci USA</em> <strong>106</strong>: 20240–20245.</p>
<p>Froelich CA, Kang S, Epling LB, Bell SP, Enemark EJ. 2014. A conserved MCM single-stranded DNA binding element is essential for replication initiation. <em>eLife</em> <strong>3</strong>: e01993.</p>
<p>Fu YV, Yardimci H, Long DT, Ho TV, Guainazzi A, Bermudez VP, Hurwitz J, van Oijen A, Schärer OD, Walter JC. 2011. Selective bypass of a lagging strand roadblock by the eukaryotic replicative DNA helicase. <em>Cell</em> <strong>146</strong>: 931–941.</p>
<p>Gambus A, Jones RC, Sanchez-Diaz A, Kanemaki M, van Deursen F, Edmondson RD, Labib K. 2006. GINS maintains association of Cdc45 with MCM in replisome progression complexes at eukaryotic DNA replication forks. <em>Nat Cell Biol</em> <strong>8</strong>: 358–366.</p>
<p>Georgescu R, Yuan Z, Bai L, de Luna Almeida Santos R, Sun J, Zhang D, Yurieva O, Li H, O'Donnell ME. 2017. Structure of eukaryotic CMG helicase at a replication fork and implications to replisome architecture and origin initiation. <em>Proc Natl Acad Sci USA</em>.</p>
<p>Gibb B, Ye LF, Gergoudis SC, Kwon Y, Niu H, Sung P, Greene EC. 2014. Concentration-dependent exchange of replication protein A on single-stranded DNA revealed by single-molecule imaging. <em>PLoS ONE</em> <strong>9</strong>: e87922.</p>
<p>Gomes XV, Gary SL, Burgers PM. 2000. Overproduction in Escherichia coli and characterization of yeast replication factor C lacking the ligase homology domain. <em>J Biol Chem</em> <strong>275</strong>: 14541–14549.</p>
<p>Heller RC, Kang S, Lam WM, Chen S, Chan CS, Bell SP. 2011. Eukaryotic origin-dependent DNA replication in vitro reveals sequential action of DDK and S-CDK kinases. <em>Cell</em> <strong>146</strong>: 80–91.</p>
<p>Homesley L, Lei M, Kawasaki Y, Sawyer S. 2000. Mcm10 and the MCM2–7 complex interact to initiate DNA synthesis and to release replication factors from origins. <em>Genes Dev</em> <strong>14</strong>: 913–926.</p>
<p>Ilves I, Petojevic T, Pesavento JJ, Botchan MR. 2010. Activation of the MCM2-7 helicase by association with Cdc45 and GINS proteins. <em>Mol Cell</em> <strong>37</strong>: 247–258.</p>
<p>Kang S, Warner MD, Bell SP. 2014. Multiple functions for Mcm2-7 ATPase motifs during replication initiation. <em>Mol Cell</em> <strong>55</strong>: 655–665.</p>
<p>Kanke M, Kodama Y, Takahashi TS, Nakagawa T, Masukata H. 2012. Mcm10 plays an essential role in origin DNA unwinding after loading of the CMG components. <em>EMBO J</em> <strong>31</strong>: 2182–2194.</p>
<p>Karnani N, Dutta A. 2011. The effect of the intra-S-phase checkpoint on origins of replication in human cells. <em>Genes Dev</em> <strong>25</strong>: 621–633.</p>
<p>Langston LD, O'Donnell M. 2008. DNA polymerase delta is highly processive with proliferating cell nuclear antigen and undergoes collision release upon completing DNA. <em>J Biol Chem</em> <strong>283</strong>: 29522–29531.</p>
<p>Lee C, Liachko I, Bouten R, Kelman Z, Tye BK. 2010. Alternative mechanisms for coordinating polymerase alpha and MCM helicase. <em>Mol Cell Biol</em> <strong>30</strong>: 423–435.</p>
<p>Li N, Zhai Y, Zhang Y, Li W, Yang M, Lei J, Tye B-K, Gao N. 2015. Structure of the eukaryotic MCM complex at 3.8 Å. <em>Nature</em> <strong>524</strong>: 186–191.</p>
<p>Merchant AM, Kawasaki Y, Chen Y, Lei M, Tye BK. 1997. A lesion in the DNA replication initiation factor Mcm10 induces pausing of elongation forks through chromosomal replication origins in Saccharomyces cerevisiae. <em>Mol Cell Biol</em> <strong>17</strong>: 3261–3271.</p>
<p>Miller JM, Arachea BT, Epling LB, Enemark EJ. 2014. Analysis of the crystal structure of an active MCM hexamer. <em>eLife</em> <strong>3</strong>: e03433.</p>
<p>Moyer SE, Lewis PW, Botchan MR. 2006. Isolation of the Cdc45/Mcm2-7/GINS (CMG) complex, a candidate for the eukaryotic DNA replication fork helicase. <em>Proc Natl Acad Sci USA</em> <strong>103</strong>: 10236–10241.</p>
<p>Muramatsu S, Hirai K, Tak Y-S, Kamimura Y, Araki H. 2010. CDK-dependent complex formation between replication proteins Dpb11, Sld2, Pol (epsilon}, and GINS in budding yeast. <em>Genes Dev</em> <strong>24</strong>: 602–612.</p>
<p>Pacek M, Tutter AV, Kubota Y, Takisawa H, Walter JC. 2006. Localization of MCM2-7, Cdc45, and GINS to the site of DNA unwinding during eukaryotic DNA replication. <em>Mol Cell</em> <strong>21</strong>: 581–587.</p>
<p>Petojevic T, Pesavento JJ, Costa A, Liang J, Wang Z, Berger JM, Botchan MR. 2015. Cdc45 (cell division cycle protein 45) guards the gate of the Eukaryote Replisome helicase stabilizing leading strand engagement. <em>Proc Natl Acad Sci USA</em> <strong>112</strong>: E249–58.</p>
<p>Prelich G, Tan CK, Kostura M, Mathews MB, So AG, Downey KM, Stillman B. 1987. Functional identity of proliferating cell nuclear antigen and a DNA polymerase-delta auxiliary protein. <em>Nature</em> <strong>326</strong>: 517–520.</p>
<p>Quan Y, Xia Y, Liu L, Cui J, Li Z, Cao Q, Chen XS, Campbell JL, Lou H. 2015. Cell-Cycle-Regulated Interaction between Mcm10 and Double Hexameric Mcm2-7 Is Required for Helicase Splitting and Activation during S Phase. <em>Cell rep</em> <strong>13</strong>: 2576–2586.</p>
<p>Remus D, Beuron F, Tolun G, Griffith JD, Morris EP, Diffley JFX. 2009. Concerted loading of Mcm2-7 double hexamers around DNA during DNA replication origin licensing. <em>Cell</em> <strong>139</strong>: 719–730.</p>
<p>Ricke RM, Bielinsky A-K. 2004. Mcm10 regulates the stability and chromatin association of DNA polymerase-alpha. <em>Mol Cell</em> <strong>16</strong>: 173–185.</p>
<p>Sun J, Evrin C, Samel SA, Fernández-Cid A, Riera A, Kawakami H, Stillman B, Speck C, Li H. 2013. Cryo-EM structure of a helicase loading intermediate containing ORC-Cdc6-Cdt1-MCM2-7 bound to DNA. <em>Nat Struct Mol Biol</em> <strong>20</strong>: 944–951.</p>
<p>Sun J, Shi Y, Georgescu RE, Yuan Z, Chait BT, Li H, O'Donnell ME. 2015. The architecture of a eukaryotic replisome. <em>Nat Struct Mol Biol</em> <strong>22</strong>: 976–982.</p>
<p>Tanaka S, Umemori T, Hirai K, Muramatsu S, Kamimura Y, Araki H. 2007. CDK-dependent phosphorylation of Sld2 and Sld3 initiates DNA replication in budding yeast. <em>Nature</em> <strong>445</strong>: 328–332.</p>
<p>Ticau S, Friedman LJ, Champasa K, Corrêa IRJ, Gelles J, Bell SP. 2017. Mechanism and timing of Mcm2-7 ring closure during DNA replication origin licensing. <em>Nat Struct Mol Biol</em> In Press.</p>
<p>Ticau S, Friedman LJ, Ivica NA, Gelles J, Bell SP. 2015. Single-molecule studies of origin licensing reveal mechanisms ensuring bidirectional helicase loading. <em>Cell</em> <strong>161</strong>: 513–525.</p>
<p>van Deursen F, Sengupta S, De Piccoli G, Sanchez-Diaz A, Labib K. 2012. Mcm10 associates with the loaded DNA helicase at replication origins and defines a novel step in its activation. <em>EMBO J</em> <strong>31</strong>: 2195–2206.</p>
<p>Watase G, Takisawa H, Kanemaki MT. 2012. Mcm10 plays a role in functioning of the eukaryotic replicative DNA helicase, Cdc45-Mcm-GINS. <em>Curr Biol</em> <strong>22</strong>: 343–349.</p>
<p>Wohlschlegel JA, Dhar SK, Prokhorova TA, Dutta A, Walter JC. 2002. Xenopus Mcm10 binds to origins of DNA replication after Mcm2-7 and stimulates origin binding of Cdc45. <em>Mol Cell</em> <strong>9</strong>: 233–240.</p>
<p>Yardimci H, Wang X, Loveland AB, Tappin I, Rudner DZ, Hurwitz J, van Oijen AM, Walter JC. 2012. Bypass of a protein barrier by a replicative DNA helicase. <em>Nature</em> <strong>492</strong>: 205–209.</p>
<p>Yeeles JTP, Deegan TD, Janska A, Early A, Diffley JFX. 2015. Regulated eukaryotic DNA replication origin firing with purified proteins. <em>Nature</em> <strong>519</strong>: 431–435.</p>
<p>Yuan Z, Bai L, Sun J, Georgescu R, Liu J, O'Donnell ME, Li H. 2016. Structure of the eukaryotic replicative CMG helicase suggests a pumpjack motion for translocation. <em>Nat Struct Mol Biol</em> <strong>23</strong>: 217–224.</p>
<p>Zegerman P, Diffley JFX. 2007. Phosphorylation of Sld2 and Sld3 by cyclin-dependent kinases promotes DNA replication in budding yeast. <em>Nature</em> <strong>445</strong>: 281–285.</p>
<p><strong>FIGURE LEGENDS</strong></p>
<p><strong>Figure 1. Mcm10 binds to a highly-conserved region of Mcm2</strong></p>
<ol style="list-style-type: upper-alpha">
<li><blockquote>
<p>Mcm2/4/6 copurifies with Mcm10. Purified Mcm2-7 (lane 1) or indicated eluates during Mcm10-FLAG purification (lane 2-4) were separated by SDS-PAGE and stained with Coomassie. Treatment of the anti-FLAG eluate with λ-phosphatase resolved three proteins in an equimolar ratio, two of which co-migrated with Mcm4 and Mcm6. The middle protein migrated more slowly after λ-phosphatase treatment, a characteristic of Mcm2 dephosphorylation.</p>
</blockquote></li>
<li><blockquote>
<p>Mcm10 preferentially binds Mcm2. Purified Mcm10-FLAG was incubated with individual purified Mcm subunits followed by anti-FLAG immunoprecipitation (IP) and separated by SDS-PAGE and stained with Krypton (lanes 6-10). Control IPs lacking Mcm10-FLAG (lane 1-5), the equivalent amounts of Mcm2-7 subunits added to the IPs (lanes 11-15) and purified Mcm2-7 (lane 16) were separated on the same gel.</p>
</blockquote></li>
<li><blockquote>
<p>Diagram of Mcm2 domain structure and the truncations used in this study. For each truncated protein, the included amino acids and epitope tag used for purification are indicated.</p>
</blockquote></li>
<li><blockquote>
<p>Mcm10 binding requires the linker region between the A-subdomain and the OB-fold of Mcm2. Purified Mcm2 truncations were tested for co-IP with FLAG-Mcm10-V5 followed by separation by SDS-PAGE and Coomassie staining (lanes 11-20). The equivalent amounts of the Mcm2 truncation proteins added to the co-IP experiments were similarly analyzed (lanes 1-10).</p>
</blockquote></li>
<li><blockquote>
<p>The Mcm10-binding motif on Mcm2 is conserved across all eukaryotes but not in other Mcm2-7 subunits. (i) Alignment of the Mcm10-binding motif of Mcm2 for <em>Saccharomyces cerevisiae (S. cer), Schizosaccharomyces pombe (S. pom), Drosophila melanogaster (D. mel), Xenopus laevis (X. lae), Mus musculus (M. mus), Homo sapiens (H. sap).</em> (ii) Alignment of the Mcm10-binding motif of <em>Saccharomyces cerevisiae</em> Mcm2-7 subunits. Homology between the Mcm10-binding motif in Mcm2 and Mcm4 is indicated.</p>
</blockquote></li>
<li><blockquote>
<p>The Mcm10-binding motif of Mcm2 is essential. In all strains, the endogenous <em>MCM2</em> gene is deleted and a copy of WT <em>MCM2</em> is present on a <em>URA3</em>-containing plasmid. <em>MCM2</em> mutants that eliminated (<em>mcm2-Δ290-299</em>) or mutated (<em>mcm2-mbm</em>) the Mcm10-binding motif were integrated into the <em>LEU2</em> locus. Growth on ‑URA media retains WT <em>MCM2</em> and indicates that the mutants are not dominant. Growth on 5-FOA selects against cells containing WT <em>MCM2</em> plasmid revealing the functionality of <em>mcm2-Δ290-299</em> or <em>mcm2-mbm</em> alleles. Five-fold serial dilutions of cells were grown on the indicated media for 3 days at 30°C.</p>
</blockquote></li>
</ol>
<blockquote>
<p>See also Table S1.</p>
</blockquote>
<p><strong>Figure 2. Identification of Mcm10-bypass alleles of Mcm2</strong></p>
<blockquote>
<p>(A) The Mcm10-binding motif of Mcm2 is buried in the absence of Mcm10. (Left) The cryo-EM structure of the CMG complex (PDB: 3JC5, Yuan et al., 2016). (Right) Space-filling representation of Cdc45, Mcm2 A-subdomain and OB-fold. Residues mutated in the Mcm2-mbm mutant are shown in red.</p>
<p>(B) Ribbon diagram of the Mcm2 A-subdomain (cyan) and OB-fold domain (blue). The residues predicted to be involved in the A-subdomain/OB-fold interaction and mutated in (C) and (D) are also shown.</p>
<p>(C) Viability of <em>MCM2</em> mutants predicted to disrupt the A-subdomain/OB-fold interaction. The indicated <em>MCM2</em> mutants were tested for complementation of a <em>MCM2</em> deletion (5 FOA). Growth on CSM media retains WT <em>MCM2</em>. Five-fold serial dilutions of cells were grown on the indicated media for 3 days at 30°C.</p>
<p>(D) <em>mcm2-bom1</em> and <em>mcm10-bom2</em> bypass the lethal depletion of Mcm10-FRB. Genetic complementation of the Mcm10 anchor-away phenotype by indicated alleles of <em>MCM2</em> or <em>MCM10</em>. Cells were spotted and grown as in (B).</p>
<p>See also Figure S1 and S6.</p>
</blockquote>
<p><strong>Figure 3. Mcm10 addition stabilizes the CMG complex</strong></p>
<ol style="list-style-type: upper-alpha">
<li><blockquote>
<p>Mcm10 preferentially associates with CMG complexes. Reaction scheme for CMG formation assay (left). The indicated purified proteins were sequentially incubated with <em>ARS1</em>-containing 3.7 kb plasmids coupled to magnetic beads. The previous reaction mix was removed prior to addition of the next without washing the beads. DNA beads or the indicated DNA-associated complexes formed at the end of each incubation were incubated with the indicated amount of Mcm10 for 1 hour (right). Bead-associated proteins were washed with low-salt buffer and detected by immunoblot.</p>
</blockquote></li>
<li><blockquote>
<p>Mcm10-dependent formation of salt-stable CMG complexes. CMG formation was performed as in (A) except after the final incubation the reactions were washed with low-salt (LSW; 0.3 M KGlut) or high-salt (HSW; 0.5 M NaCl) containing buffers. Assays were performed in the presence and absence of Mcm10 or DDK as indicated (right). Omission of DDK was used as a control for non-specific DNA binding of Cdc45, GINS, RPA and Pol <strong>ε</strong> (Mcm2-7 loading is DDK-independent).</p>
</blockquote></li>
<li><blockquote>
<p>Salt-stable CMG complexes are competent for DNA replication. The reaction scheme is illustrated (left). After CMG formation, as in (A), the DNA beads were washed with the indicated buffer followed by addition of the indicated proteins and [α-<sup>32</sup>P]-dCTP. Where indicated, Mcm10 and DDK were omitted during both CMG formation and DNA replication. Replication products were separated on a 1% alkaline agarose gel and imaged using a phosphorimager.</p>
</blockquote></li>
<li><blockquote>
<p>Mutants in the Mcm2 A-subdomain and OB-fold domain are defective for Mcm10 binding and CMG formation. CMG formation assays were performed with Mcm2-7<sup>WT</sup> or Mcm2-7 including Mcm2-mbm (Mcm2-7<sup>2-mbm</sup>), Mcm2-bom1 (Mcm2-7<sup>2-bom1</sup>), or Mcm2-bom2 (Mcm2-7<sup>2-bom2</sup>). All reactions were washed with low-salt buffer.</p>
</blockquote></li>
<li><blockquote>
<p>Mcm2-7<sup>2-mbm</sup> is defective for DNA replication. The reaction scheme was the same as (A), except the indicated replication proteins were included in the final step to allow DNA replication initiation. DNA replication products were monitored as in (C).</p>
</blockquote></li>
<li><p>Mcm2-7<sup>2-bom1</sup> and Mcm2-7<sup>2-bom2</sup> bypass Mcm10 function <em>in vitro</em>. Mcm2-7<sup>WT</sup>, Mcm2-7<sup>2-bom1</sup> or Mcm2-7<sup>2-bom2</sup> were tested for their ability to participate in DNA replication <em>in vitro</em>. Assays were performed with and without DDK and Mcm10 as indicated. DNA replication assays were performed and replication products analyzed as described in (E).</p>
<p>See also Figure S2 and S3.</p></li>
</ol>
<p><strong>Figure 4. Mcm10 promotes replication elongation</strong></p>
<ol style="list-style-type: upper-alpha">
<li><blockquote>
<p>Decreased Mcm10 leads to shorter replication products. Replications reactions were performed as described in Fig. 3E with the indicated amounts of Mcm10. Replication product intensities are plotted with the colors corresponding to the box above the given lane. Red lines indicate the midpoint of the top 10% of replication product intensity for a given lane.</p>
</blockquote></li>
<li><blockquote>
<p>Low concentrations of Mcm10 are competent for CMG activation. The indicated amount of WT Mcm10 (lane 3-7) was used for (i) CMG formation followed by a low-salt wash or (ii) CMG formation followed by a high-salt wash. The reaction scheme for (i) and (ii) was the same as Fig. 3A. Immunoblots of DNA-associated proteins are shown.</p>
</blockquote></li>
<li><blockquote>
<p>Titration of Mcm10 alters replication-product lengths in the absence of PCNA. DNA replication reactions were performed as described in Fig. 3E with the indicated amounts of PCNA. Replication product distributions for lanes 3-5 were analyzed as described in (A).</p>
</blockquote></li>
<li><blockquote>
<p>Mcm10 stimulates replication elongation. High-salt-resistant CMG complexes were formed and DNA replication was initiated with the same proteins as in Fig. 3C except except Mcm10 was either omitted or the indicated amount was added during the replication step. Replication product intensities were analyzed as described in (A).</p>
</blockquote></li>
</ol>
<blockquote>
<p>See also Figure S4.</p>
</blockquote>
<p><strong>Figure 5. Identification of biologically important regions of Mcm10</strong></p>
<ol style="list-style-type: upper-alpha">
<li><blockquote>
<p>Schematic of the Mcm10 protein and the mutants analyzed. Mcm10 domain organization is shown above a set of truncation mutants tested for Mcm10 complementation (included Mcm10 amino acids are indicated). All proteins included a FLAG tag at the C-terminus and all C-terminal truncations include the <em>ORC2</em> nuclear-localization sequence (NLS). For residues 399-434, six alanine-scanning mutations were constructed in the context of full-length Mcm10.</p>
</blockquote></li>
<li><blockquote>
<p>Mcm10 mutant complementation. (Top) Genetic complementation scheme. Addition of rapamycin results in depletion of Mcm10-FRB from the nucleus. (Bottom) Genetic complementation of <em>mcm10-FRB</em> anchor-away phenotype with the indicated <em>MCM10</em> alleles inserted to <em>LEU2</em> locus. Five-fold serial dilutions of were spotted on the indicated media and incubated at 30<sup>o</sup>C for 3 days.</p>
</blockquote></li>
</ol>
<blockquote>
<p>See also Figure S1.</p>
</blockquote>
<p><strong>Figure 6. Mcm10-A3 is defective for stimulating replication elongation</strong></p>
<ol style="list-style-type: upper-alpha">
<li><blockquote>
<p>Mcm10-A3 is competent for CMG activation. The indicated amount of WT Mcm10 (lane 3) or Mcm10-A3 (lanes 4-7) was used for (i) CMG formation followed by a low-salt wash, (ii) CMG formation followed by a high-salt wash, or (iii) DNA replication. The reaction scheme for (i) and (ii) was the same as Fig. 3A. The reaction scheme for (iii) was the same as Fig. 3E. Immunoblots of DNA-associated proteins are shown for (i) and (ii). Labeled DNA replication products were analyzed as described in Fig. 3C for (iii).</p>
</blockquote></li>
<li><blockquote>
<p>Mcm10 but not Mcm10-A3 facilitates DNA replication after CMG formation. High-salt-resistant CMG complexes were formed with the indicated Mcm10 protein. DNA replication was initiated as in Fig. 3C except WT Mcm10, Mcm10-A3 or no Mcm10 was included as indicated. DNA replication products were analyzed as in Fig. 3C.</p>
</blockquote></li>
<li><blockquote>
<p>Mcm10 facilitates replication elongation <em>in vivo</em>. Flow cytometry analysis of the DNA content of <em>cdc7-1</em> cells with indicated alleles of <em>MCM10</em>. Cells, grown in YP-glucose, were first arrested in G1 phase with α-factor and then in early S phase with hydroxyurea (HU) at 25<sup>o</sup>C. Next, <em>mcm10-1td</em> was degraded by shifting cells to 37<sup>o</sup>C in HU. Subsequently, cells were released from HU arrest at 37<sup>o</sup>C into nocodozole-containing media at 37<sup>o</sup>C . To test the complementation of the <em>mcm10-1td</em> allele, an additional copy of <em>MCM10</em> or <em>mcm10-A3</em> was inserted at the <em>LEU2</em> locus.</p>
</blockquote></li>
</ol>
<blockquote>
<p>See also Figure S1 and S5.</p>
</blockquote>
